source/Program.cs

```diff
@@ -30,9 +30,9 @@ public static async Task Main(string[] args)
                             Port = 9090,
                             UseDefaultCollectors = false
                         }));
-                    services.AddHostedService<TeamCityQueueLengthScraper>();
-                    services.AddHostedService<TeamCityBuildScraper>();
-                    services.AddHostedService<TeamCityBuildArtifactScraper>();
+                    // services.AddHostedService<TeamCityQueueLengthScraper>();
```

Comment:
Pretty sure you didn't mean to include this :)

Reply:
You would be correct! All done 😅